commit 738ee4d006c3fbce6f0a4d26849e10abe37f184a (refs/changes/54/4254/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-18T12:21:02-07:00 (1 year, 4 months ago)
    
    backout MANTA-3719 Need Firewall Rules for Postgres (needs more work)

diff --git a/boot/setup.sh b/boot/setup.sh
index c892924..49b403a 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -77,9 +77,6 @@ function common_enable_services {
 
     svccfg import /opt/smartdc/manatee/smf/manifests/sitter.xml
     # For SDC we'll let configure decide if it wants to enable sitter or not.
-
-    echo "Starting waferlock"
-    svccfg import /opt/smartdc/waferlock/smf/manifests/waferlock.xml
 }
 
 function common_manatee_setup {
@@ -178,7 +175,7 @@ function add_manatee_profile_functions {
 
 
 # Local manifests
-CONFIG_AGENT_LOCAL_MANIFESTS_DIRS="/opt/smartdc/$role /opt/smartdc/waferlock"
+CONFIG_AGENT_LOCAL_MANIFESTS_DIRS=/opt/smartdc/$role
 
 # Include common utility functions (then run the boilerplate)
 source /opt/smartdc/boot/lib/util.sh
@@ -192,7 +189,6 @@ add_manatee_profile_functions
 sdc_log_rotation_add manatee-sitter /var/svc/log/*manatee-sitter*.log 1g
 sdc_log_rotation_add manatee-snapshotter /var/svc/log/*manatee-snapshotter*.log 1g
 sdc_log_rotation_add manatee-backupserver /var/svc/log/*manatee-backupserver*.log 1g
-sdc_log_rotation_add waferlock /var/svc/log/*waferlock*.log 1g
 sdc_log_rotation_add manatee-postgres /var/pg/postgresql.log 1g
 sdc_log_rotation_setup_end
 
